### ISSUE TITLE
[popup_trigger] accepts existing popup as selection option in 'Targeted Popup' input

### ISSUE THREAD

**Issue by rgadon107** [PopupMaker/Popup-Maker#508]
When using the [popup_trigger] shortcode on an existing popup ('popup_1') to set a trigger to display 'popup_2', the 'Popup Settings' box -> 'Targeted Popup' selector accepts that same popup ('popup_1')as an option. The plugin should be changed to not display, nor accept that popup option ('popup_1') as a trigger input. 

![popup trigger -- general tab targeted popup option settings](https://user-images.githubusercontent.com/3718603/36751642-5b6f1572-1bc6-11e8-84a8-c12aea10a8bb.png)

The yellow arrow points to the 'Text Test Popup' option as an accepted trigger input. That's the same popup in which the trigger is being set. The only acceptable options should be either 'Form Test Popup' (red arrow) or the 'Custom' option.

**Comment by rgadon107** [comment]
@danieliser I now understand how the 'Targeted Popup' option in the 'Popup Settings' box works. In the case of a single, published popup, the option setting displays 2 selections; the published popup, and an option of id='custom'. The contextual description below the option setting doesn't provide users with adequate information on which option to select. Once I understood the _intent_ and application of the 'custom' option, I realized that users have an option by which to set a second trigger.  I addressed the decision logic of when and which option to choose in the 'Targeted Popup' setting within the documentation article https://docs.wppopupmaker.com/article/90-shortcode-popup-trigger. 

Perhaps an improvement for either a version 1.7.x or 1.8 milestone would be to add a 'doclink' to the option setting, and change the contextual description below the option selection to clarify which option to choose.
